release-notes/opensearch-dashboards.release-notes-1.3.8.md

```diff
@@ -0,0 +1,32 @@
+# Version 1.3.8 Release Notes
+
+
+### 🛡 Security
+
+- [CVE-2022-25901] Bump supertest ([#3326](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3326))
```

Comment:
For consistency, let's add the from/to version numbers

Reply:
Change it to ` Bump supertest from 2.0.5 to 2.0.12`